core-bundle/src/EventListener/DataContainer/RecordPreviewListener.php

```diff
@@ -41,6 +41,10 @@ public function registerDeleteCallbacks(string $table): void
             return;
         }
 
+        if ($GLOBALS['TL_DCA'][$table]['config']['dataContainer'] === 'Folder') {
```

Comment:
```suggestion
        if ($GLOBALS['TL_DCA'][$table]['config']['dataContainer'] !== 'Table') {
```

I would skip all data containers that are not `DC_Table`

Reply:
That would also exclude `DC_General`, which could be supported.

Reply:
Everyone with a custom DC could still register the callbacks themselves. IMHO we should be defensive here.

Reply:
See b9f16f7